app/views/contexts/hub_context.py

```diff
@@ -102,19 +113,37 @@ def get_section_url(section_id, list_item_id) -> str:
 
         return url_for('questionnaire.get_section', section_id=section_id)
 
-    def _get_row_for_section(self, section_name, section_id, list_item_id=None):
+    def _get_row_for_repeating_section(self, section_title, section_id, list_item_id):
+        repeating_title = self._schema.get_repeating_title_for_section(section_id)
+
+        placeholder_renderer = PlaceholderRenderer(
+            language=self._language,
+            schema=self._schema,
+            answer_store=self._answer_store,
+            metadata=self._metadata,
+            list_item_id=list_item_id,
+        )
```

Comment:
We should consider initialising the placeholder renderer once, rather than for every repeating section title. 

Probably a separate card, but we should also consider supporting placeholders in section titles that don't repeat.

Reply:
I've added a property for the renderer, and moved list_item_id into the render call to allow us to initialise once.